ote/src/clj/ote/services/transport.clj

```diff
@@ -757,17 +765,16 @@
         (authorization/with-group-check db user ckan-group-id
           #(operator-users-response db ckan-group-id)))
 
-      (POST "/transport-operator/:ckan-group-id/users"
-            {{:keys [ckan-group-id]}
+      (POST "/transport-operator/:ckan-group-id/:authority/users"
```

Comment:
Because client cannot be relied upon on security matters, it seems dangerous for service to rely on client on if the group or user are authority-ones. 
- Service should  use the same API as for adding users to regular groups, and resolve if it's an authority group  internally (`users/transit-authority-group-id`).
Note also how /pre-notices/authority-group-id may be invoked by a non-authority client.
- The endpoint or `manage-adding-users-to-operator` should enforce that only authorized users are allowed to add users to the authority group.

Reply:
I'm not sure if it is more dangerous because users can't invite other users to be part of operator was it a normal operator or authority.
But I changed the backend url back where it was and added db query to find out if operator is authority operator.

Reply:
Okay, in that case...